learn/cookbooks/postman_collection.md

```diff
@@ -19,11 +19,11 @@ Once you have downloaded the [Postman collection](/postman/meilisearch-collectio
 
 ![Selecting "Edit" from the overflow menu](/postman/edit.png)
 
-Set the token if needed (set to `masterKey` by default):
+Set the "Token" if needed (set to `masterKey` by default):
 
 ![The "Token" field set to masterKey and "Type" to Bearer Token in the "Authorization" tab.](/postman/set_token.png)
 
-Set the host URL (set to Meilisearch's local port by default) and the index UID (set to `indexUID` by default):
+Set the "url" (set to Meilisearch's local port by default) and the "indexUID" (set to `indexUID` by default):
```

Comment:
This is a tricky edge-case for our style rules, since it is both explicitly a variable, and some text in a GUI. I chose backticks because it will effectively be used as a variable by the program. But either backticks or quotes would be fine—as long as it's consistent with the following line (which currently uses backticks).

```suggestion
Set `url` (set to Meilisearch's local port by default) and `indexUID` (set to `indexUID` by default):
```